README.md

```diff
@@ -22,9 +22,62 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
 
 The Presto Foundation maintains a [public calendar](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_vrjlva5b0u73ps75fvnv5sasi4%40group.calendar.google.com&ctz=America%2FChicago) for TSC meetings. These meetings are open for anyone to join.
 
+Because we work in a highly distributed environment and will rarely meet in person, participants are encouraged to use video as appropriate.
 
 ## Members
 
-The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers), and the current TSC chair is Nezih Yigitbasi ([@nezihyigitbasi](https://github.com/nezihyigitbasi)). At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project’s code repository
+The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/.github/blob/master/CONTRIBUTING.md).
 
-https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md
+The current members of the Presto TSC are:
+
+| Name                      | Github                                              | Affiliation | Term begins  | Term ends    |
+| ------------------------- | --------------------------------------------------- | ------------|--------------|--------------|
```

Comment:
This list is not up-to-date. It misses Leiqing Cai. Please see https://github.com/prestodb/presto/wiki/committers.

Reply:
Thanks, I was working from an old list.  This is updated now.